github-repos/github-repos.Rmd

```diff
@@ -0,0 +1,80 @@
+---
+title: "GitHub repo details"
+output:
+  html_document:
+    code_folding: hide
+    toc: true
+    theme: united
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  cache = TRUE
+)
+```
+
+```{r get_repos}
+
```

Comment:
I think you need to load the package here?
`library(pacta.sit.rep)`

Reply:
I've been using `devtools::load_all()` and rendering to html locally. 

Longer-term I think we would be better off putting any RMD templates in `inst/extdata`, and creating a `generate_sitrep` function that calls those templates with `system.file`, and that wshould handle all the library issues.

Reply:
Happy to do that, but then this becomes a MUCH bigger project than I had initially envisioned for the `pacta.sit.rep` repo. (Not strictly against that, just a comment worth considering). 

The goal I had in mind for this repo was to be focused on communicating the state of our code to non-tech folks that are interested (e.g. Nayra, SLM, hodie, etc.). And also to say: hey here's a link to the most recently generated report. 

The functionality you suggest is super interesting, but definitely more developer focused. 

Maybe let's have a call to align on this (I don't think there's any reason that this repo can't support both use-cases, just want to make sure we are aligned)

Reply:
Yeah, mostly I'm looking at the dev-focused stuff as related and naturally fitting here, rather than a separate repo, but I'm not worrying about making everythign "clean" for it yet.